info/available-rules.md

```diff
@@ -1,131 +1,131 @@
-| 5    | 5.2.6    | CUSTOM_LABEL                              | Check: that using unnecessary, custom label                                                                                                                                                                                                                  | no  | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
-| 5    | 5.2.7    | PARAMETER_NAME_IN_OUTER_LAMBDA            | Check: warns if outer lambda uses implicit parameter `it`                                                                                                                                                                                                    | no  | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
+| 4    | 4.2.2    | TYPE_ALIAS                                | Check: if a type reference for property is longer than expected                                                                                                                                                                                                 | yes | typeReferenceLength                                                                                                                                                                                                        | -                                                                                                                                                                                                                             |
```

Comment:
Duplicated line. Should we remove it?                                                                                                         |
| 4    | 4.2.2    | TYPE_ALIAS                                | Check: if the type reference for a property is longer than expected                                                                                                                                                                                                 | yes | typeReferenceLength

Reply:
Yes, please.

Thanks for spotting this.

Reply:
OK